### ISSUE TITLE
Build failure with GCC 13 due to missing <cstdint> include

### ISSUE THREAD

**Issue by qookei** [MusicPlayerDaemon/MPD#1717]
### Describe the bug

GCC 13 removed a transient include of `<cstdint>` from (I think?) `<iostream>` which prevents mpd from building with it.

The build fails with the following error:
```
FAILED: src/io/libio.a.p/FileReader.cxx.o 
x86_64-pc-linux-gnu-g++ -Isrc/io/libio.a.p -Isrc/io -I../mpd-0.23.9/src/io -Isrc -I../mpd-0.23.9/src -I. -I../mpd-0.23.9 -fdiagnostics-color=always -Wall -Winvalid-pch -Wnon-virtual-dtor -Wextra -Wpedantic -std=c++17 -ffast-math -ftree-vectorize -Wcast-qual -Wdouble-promotion -Wmissing-declarations -Wshadow -Wunused -Wvla -Wwrite-strings -Wno-stringop-overflow -fno-threadsafe-statics -fmerge-all-constants -Wcomma-subscript -Wextra-semi -Wmismatched-tags -Woverloaded-virtual -Wsign-promo -Wvolatile -Wvirtual-inheritance -Wno-non-virtual-dtor -Wsuggest-override -fvisibility=hidden -ffunction-sections -fdata-sections -D_GNU_SOURCE -O2 -march=znver2 -pipe -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -fPIC -MD -MQ src/io/libio.a.p/FileReader.cxx.o -MF src/io/libio.a.p/FileReader.cxx.o.d -o src/io/libio.a.p/FileReader.cxx.o -c ../mpd-0.23.9/src/io/FileReader.cxx
In file included from ../mpd-0.23.9/src/io/FileReader.cxx:30:
../mpd-0.23.9/src/io/FileReader.hxx:97:9: error: ‘uint64_t’ does not name a type
   97 |         uint64_t GetSize() const noexcept {
      |         ^~~~~~~~
../mpd-0.23.9/src/io/FileReader.hxx:43:1: note: ‘uint64_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
   42 | #include "io/UniqueFileDescriptor.hxx"
  +++ |+#include <cstdint>
   43 | #endif
../mpd-0.23.9/src/io/FileReader.hxx:109:9: error: ‘uint64_t’ does not name a type
  109 |         uint64_t GetPosition() const noexcept {
      |         ^~~~~~~~
```

## Version
mpd-0.23.9

**Comment by MaxKellermann** [comment]
Your MPD version is old and unsupported.

**Comment by qookei** [comment]
Sorry, didn't notice. I just tried 0.23.11 and it still fails in the same place:
```
../mpd-0.23.11/src/io/FileReader.hxx:97:9: error: ‘uint64_t’ does not name a type
   97 |         uint64_t GetSize() const noexcept {
      |         ^~~~~~~~
../mpd-0.23.11/src/io/FileReader.hxx:43:1: note: ‘uint64_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
   42 | #include "io/UniqueFileDescriptor.hxx"
  +++ |+#include <cstdint>
   43 | #endif
../mpd-0.23.11/src/io/FileReader.hxx:109:9: error: ‘uint64_t’ does not name a type
  109 |         uint64_t GetPosition() const noexcept {
      |         ^~~~~~~~
```

**Comment by qookei** [comment]
For future reference this was fixed in f6f875133245368ffb985a9cf2fe00130d755cb4.
